crates/bevy_pbr/src/environment_map/mod.rs

```diff
@@ -51,6 +51,10 @@ impl Plugin for EnvironmentMapPlugin {
 pub struct EnvironmentMapLight {
     pub diffuse_map: Handle<Image>,
     pub specular_map: Handle<Image>,
+    /// Scale factor applied to both the diffuse and specular cubemap.
+    /// After applying this multiplier to the image samples, the resulting values should
+    /// be in units of [cd/m^2](https://en.wikipedia.org/wiki/Candela_per_square_metre).
```

Comment:
Someone should double check this. $cd/m^2$ is a unit of luminance, but I'm not sure that's actually what the diffuse and specular map store? I've seen some things that suggesting it might be illuminance (units $cd*sr/m^2$), or perhaps something else

Reply:
This doc comment is a bit misleading tbh. The scale factor is applied to the calculated light, not the cubemap values themselves.

Reply:
cd/m^2 seems correct according to filament https://google.github.io/filament/Filament.html#lighting/imagebasedlights/iblunit